Library/ViewModels/ManageViewPledgeViewModelTests.swift

```diff
@@ -110,4 +120,64 @@ internal final class ManageViewPledgeViewModelTests: TestCase {
 
     self.showActionSheetMenuWithOptions.assertValues([[.contactCreator]])
   }
+
+  func testGoToCancelPledge() {
+    self.vm.inputs.configureWith(.template, reward: .template)
+    self.vm.inputs.viewDidLoad()
+
+    self.goToCancelPledge.assertDidNotEmitValue()
+
+    self.vm.inputs.menuButtonTapped()
+    self.vm.inputs.menuOptionSelected(with: .cancelPledge)
+
+    self.goToCancelPledge.assertValueCount(1)
+  }
+
+  func testGoToChangePaymentMethod() {
+    self.vm.inputs.configureWith(.template, reward: .template)
+    self.vm.inputs.viewDidLoad()
+
+    self.goToChangePaymentMethod.assertDidNotEmitValue()
+
+    self.vm.inputs.menuButtonTapped()
+    self.vm.inputs.menuOptionSelected(with: .changePaymentMethod)
+
+    self.goToChangePaymentMethod.assertValueCount(1)
+  }
+
+  func testGoToContactCreator() {
+    self.vm.inputs.configureWith(.template, reward: .template)
+    self.vm.inputs.viewDidLoad()
+
+    self.goToContactCreator.assertDidNotEmitValue()
+
+    self.vm.inputs.menuButtonTapped()
+    self.vm.inputs.menuOptionSelected(with: .contactCreator)
+
+    self.goToContactCreator.assertValueCount(1)
+  }
+
+  func testGoToRewards() {
+    self.vm.inputs.configureWith(.template, reward: .template)
+    self.vm.inputs.viewDidLoad()
+
+    self.goToRewards.assertDidNotEmitValue()
+
+    self.vm.inputs.menuButtonTapped()
+    self.vm.inputs.menuOptionSelected(with: .chooseAnotherReward)
+
+    self.goToRewards.assertValues([.template])
```

Comment:
I think for the `testGoToRewards` and `testGoToUpdatePledge`, would be more readable if the type of `.template` was explicit, like:
```
self.goToRewards.assertValues([Project.template])
```
```
self.goToUpdatePledge.assertValues([Project.template])
```
This would also avoid confusion, since the `configureWith` function is taking 2 different types of `.template`.

Other option would be to simply create variables and use them instead: 
```
let project = Project.template
let reward = Reward.template
```